rest/rest-resources/src/main/java/org/apache/brooklyn/rest/util/MultiSessionAttributeAdapter.java

```diff
@@ -638,12 +638,22 @@ public MultiSessionAttributeAdapter resetExpiration() {
         Handler[] hh = getSessionHandlers();
         if (hh!=null) {
             for (Handler h: hh) {
-                Session ss = ((SessionHandler)h).getSession(getId());
-                if (ss!=null) {
+                Session ss = getSessionSafely(h, getId());
+                if (ss != null) {
                     ss.setMaxInactiveInterval(maxInativeInterval);
                 }
             }
         }
         return this;
     }
+
+    private static Session getSessionSafely(Handler h, String id) {
+        if (((SessionHandler)h).getSessionCache()==null) {
+            // suppress the log warning that the call to getSession can trigger, if racing during startup
+            log.debug("Skipping session reset expiration for "+id+" on "+h+" because session cache not initialized (yet)");
```

Comment:
The logger should allow to parametrise the message like the following:
 `           log.debug("Skipping session reset expiration for {} on {} because session cache not initialized (yet)", id, h);`

Reply:
The logger does allow that, but (personal preference) I find that decreases readability and leads to bugs where the order of parameters gets out of synch with the message, so I only do that where the savings in skipping the string concatenation are likely to be significant when debug mode is off.  (Or alternatively do a `log.isDebugEnabled()` check.). This block should only happen rarely during init so not worth it IMO.